lib/Documentable/Index.pm6

```diff
@@ -35,8 +34,8 @@ method new(
 
 method url() {
     my $index-text = recurse-until-str($.pod).join;
-    my @indices    = $.pod.meta;
-    my $fragment = qq[index-entry{@indices ?? '-' !! ''}{@indices.join('-')}{$index-text ?? '-' !! ''}$index-text]
+    my $indices    = $.pod.meta[0];
+    my $fragment = qq[index-entry{$indices ?? "-$indices[1]" !! ''}{$index-text ?? '-' !! ''}$index-text]
```

Comment:
Not entirely clear what is being done here. You're using $indices[1] === $.pod.meta[0][1] which is what?